spec/02-integration/09-hybrid_mode/09-forward-proxy_spec.lua

```diff
@@ -0,0 +1,176 @@
+local helpers = require "spec.helpers"
+local pl_path      = require "pl.path"
+local pl_file      = require "pl.file"
+
+
+local fixtures = {
+  stream_mock = {
+    forward_proxy = [[
+    server {
+      listen 16797;
+      error_log logs/proxy.log debug;
+
+      content_by_lua_block {
+        require("spec.fixtures.forward-proxy-server").connect()
+      }
+    }
+
+    server {
+      listen 16796;
+      error_log logs/proxy_auth.log debug;
+
+      content_by_lua_block {
+        require("spec.fixtures.forward-proxy-server").connect({
+          basic_auth = ngx.encode_base64("test:konghq"),
```

Comment:
Argh, it looks like I forgot to port the basic auth support for this fixture back to OSS, so I don't think this is doing anything at the moment :(

If you grab `spec/fixtures/forward-proxy-server.lua` from kong-ee:master it should be fine to drop into this branch as-is.

Reply:
i copied it in, and fixed a small typo, will let the ce->ee merge to propogate it to EE